docker-compose.yaml

```diff
@@ -1,4 +1,4 @@
-version: '3.8'
+version: '3.3'
```

Comment:
Why the change?

Reply:
ERROR: Version in "./docker-compose.yaml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/

Reply:
I can't start the db with 3.8.
docker-compose up -d db command fails

Reply:
@ekstromchristian do you have the latest docker-compose version?

https://docs.docker.com/compose/compose-file/

```
$ docker-compose --version
docker-compose version 1.27.4, build unknown
```

Reply:
docker-compose version 1.24.1, build 4667896b

Reply:
So either upgrade your local docker-compose or change to 3.3, we’re not really using any important features from the latest versions.

Reply:
I did the upgrade and added this to contribute.md